docs/guide/deployment.md

```diff
@@ -319,3 +319,90 @@ Verify your project is successfully published by Surge by visiting `myawesomepro
 
     cd -
     ```
+
+
+### Docker (Nginx)
+
+Deploying your application using nginx inside of docker can be done easily with a minor modification of the nginx configuration.
```

Comment:
It's unclear when you say _minor modification of the nginx configuration_, it would be nice if explicitly specify what we mean by _the nginx configuration_. Also, we can drop **easily**.

Reply:
I removed some of the confusing language.